manifests/repo.pp

```diff
@@ -0,0 +1,24 @@
+# Configure the Pulpcore repo
+#
+# @param version
+#   The Pulpcore version to use
+class pulpcore::repo (
+  Pattern['^\d+\.\d+$'] $version = '3.6',
+) {
+  $context = {
+    'version'   => $version,
+    'dist_code' => "el${facts['os']['release']['major']}",
```

Comment:
The term `dist_code` seemed weird, but I get the thought behind naming it such. I most often see it called a `distirbution tag` or `dist tag` in related docs, so for congruence naming this `dist_tag` would align.

Reply:
With CentOS you can use `$releasever` in yum configs and that's `7` or `8` but with RHEL it can be `7Server`. That would have been my preferred solution if RHEL didn't complicate it. For now I'll go with `dist_tag` then.